app/lib/rocketchat/services/sdk.ts

```diff
@@ -158,6 +162,22 @@ class Sdk {
 	onStreamData(...args: any[]) {
 		return this.sdk.onStreamData(...args);
 	}
+
+	login(credentials: ICredentials) {
+		return this.sdk.login(credentials);
+	}
+
+	checkAndReopen() {
+		return this.sdk.checkAndReopen();
+	}
+
+	abort() {
+		return this.sdk.abort();
+	}
+
+	connect() {
+		return this.sdk.connect();
+	}
```

Comment:
I don't think it's interesting to create a wrapper for each method of this one, since they are simple methods, better to use `sdk.current`